doc/md/guides/orms/gorm.md

```diff
@@ -0,0 +1,221 @@
+---
+id: gorm
+title: Automatic migration planning for GORM
+slug: /guides/orms/gorm
+---
+
+## TL;DR
+* [GORM](https://gorm.io) is a an ORM library that's widely used in the Go community.
+* [Atlas](https://atlasgo.io) is an open-source tool for inspecting, planning, linting and
+  executing schema changes to your database.
+* Developers using GORM can use Atlas to automatically plan schema migrations
+  for them, based on the desired state of their schema instead of crafting them by hand.
+
+## Automatic migration planning for GORM
+
+GORM is a popular ORM widely used in the Go community. GORM allows users to 
+manage their database schemas using its [AutoMigrate](https://gorm.io/docs/migration.html#Auto-Migration)
+feature, which is usually suffucient during development and in many simple cases. 
+
+However, at some point, many teams need more control and decide to employ 
+a [versioned migrations](/concepts/declarative-vs-versioned#versioned-migrations) 
+methodology. Once this happens, the responsibility for planning migration scripts and making
+sure they are in line with what GORM expects at runtime is moved to developers.
+
+Atlas can automatically plan database schema migrations for developers using GORM.
+Atlas plans migrations by calculating the diff between the _current_ state of the database,
+and it's _desired_ state.
+
+For GORM users, the current state can be thought of as the database schema that would have
+been created by GORM's [AutoMigrate](https://gorm.io/docs/migration.html#Auto-Migration)
+feature, if run on an empty database. 
+
+The desired state can be provided to Atlas via an Atlas schema [HCL file](https://atlasgo.io/atlas-schema/sql-resources) 
+or as a connection string to a database that contains the desired schema.
+
+In this guide, we will show how Atlas can automatically plan schema migrations for
+GORM users.
+
+## Prerequisites
+
+* An existing project with a GORM.
+* Docker
+* Atlas ([installation guide](https://atlasgo.io/getting-started/#installation))
+
+## Dev database
+To plan a migration from the current to the desired state, Atlas uses a [Dev Database](/concepts/dev-database),
+which is usually provided by a locally running container with an empty database of the type
+you work with (such as MySQL or PostgreSQL).
+
+To spin up a local MySQL database that will be used as a dev-database in our example, run:
+
+```text
+docker run --rm --name atlas-db-dev -d -p 3306:3306 -e MYSQL_DATABASE=dev -e MYSQL_ROOT_PASSWORD=pass mysql:8
+```
+
+As reference for the next steps, the URL for the Dev Database will be:
+```text
+mysql://root:pass@localhost:3306/dev
+```
+
+## Create an auto-migration script
+
+Depends on how your project is set up, you may be able to skip this part. 
```

Comment:
Maybe add some context? Who can skip this part? Is there some more specific info?

Reply:
i'll rm